doc/releases/changelog-dev.md

```diff
@@ -19,6 +19,37 @@
 
 <h3>Improvements 🛠</h3>
 
+* A new :func:`~.marker` function allows for easy inspection at particular points in a transform program with :func:`~.specs` and :func:`~.draw` instead of having to increment ``level`` by integer amounts.
```

Comment:
```suggestion
* A new :func:`~.marker` function allows for easy inspection at particular points in a transform program
  with :func:`~.specs` and :func:`~.drawer.draw` instead of having to increment ``level``
  by integer amounts.
```